projects/isletnet.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name_cs": "IsletNet",
+  "name_en": "IsletNet",
+  "name_de": "IsletNet",
+  "subtitle_cs": "Webová služba kvantifikující pankreatické ostrůvky",
+  "subtitle_en": "Web service quantifying pancreatic islets",
+  "subtitle_de": "Automatisierter Webdienst für die Bewertung mikroskopischer Bilder von mit Dithizon gefärbten isolierten Pankreasinseln",
+  "website": "https://isletnet.com",
+  "desc_cs": "Automatizovaná webová služba věnovaná hodnocení mikroskopických snímků izolovaných pankreatických ostrůvků barvených dithizonem. Otevřený projekt spolupráce pro usnadnění mezilaboratorní standardizace. Vestavěné nástroje umožňují uživatelům ověřovat algoritmy IsletNet. Uživatelům můžou své snímky sdílet s vývojáři pomocí zaškrtávacího políčka na obrazovce výsledků. Sdílené obrázky se používají pro další trénování neuronových sítí za účelem neustálého zlepšování služeb.",

```

Comment:
prijde mi to az moc technicky a malo sorzumitelny pro general public. reknu vic sloven